project/plugins.sbt

```diff
@@ -20,3 +20,5 @@ addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.16")
 addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.5.7")
 
 addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.5.5")
+
+libraryDependencies += "org.snakeyaml" % "snakeyaml-engine" % "2.3"
```

Comment:
I guess this won't propagate to the clients, only for the build?

Reply:
Yes, exactly. It's out private thing here :wink: